### PR TITLE
Feat/switch examples

### PR DESCRIPTION
**Pull request by sandeepgeeky**
don't accept this PR.
I've sent another one, this was having too much conflicts.

### CODE REVIEW

File: example/storybook/src/components/Feedback/Spinner/index.stories.mdx [hunk 1]
```diff
@@ -29,21 +29,24 @@ Following is the default implementation of the **Spinner** component without any
...
+Wrapper,
+Spinner,
+},
+argsType: {},
+}}
+/>
+
 </AppProvider>
 
+####
```
Comment (amars29) [inline example/storybook/src/components/Feedback/Spinner/index.stories.mdx:48]: Remove unnecessary code

File: example/storybook/src/components/Feedback/Spinner/index.stories.mdx [hunk 2]
```diff
@@ -81,6 +84,119 @@ It inherits all the properties of React Native's [ActivityIndicator](https://rea
...
+#### Color Scheme
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+        <VStack space="md" >
+      <Spinner color="#4F46E5" size="large"/>
```
Comment (amars29) [inline example/storybook/src/components/Feedback/Spinner/index.stories.mdx:156]: Use token colors in all the three spinner

File: example/storybook/src/components/Feedback/Spinner/index.stories.mdx [hunk 3]
```diff
@@ -81,6 +84,119 @@ It inherits all the properties of React Native's [ActivityIndicator](https://rea
...
+#### With no empty area color
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+       
+      <Spinner color="#4F46E5"   size="large"/>
```
Comment (amars29) [inline example/storybook/src/components/Feedback/Spinner/index.stories.mdx:184]: use token color if possible

File: example/storybook/src/components/Forms/Switch/Switch.tsx [hunk 4]
```diff
@@ -15,6 +15,7 @@ export const SwitchStory = ({
     <Wrapper>
       <Center sx={{ flex: 1 }}>
         <Switch
+          size=""
```
Comment (amars29) [inline example/storybook/src/components/Forms/Switch/Switch.tsx:18]: assign the value to switch

File: example/storybook/src/components/Forms/Switch/index.stories.mdx [hunk 5]
```diff
@@ -94,6 +94,150 @@ The following are the lists of all the libraries and packages the component reli
...
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+         <HStack space="md">
+             <Switch isChecked={true}/>
+             <Text size="sm" >Public profile</Text>
```
Comment (amars29) [inline example/storybook/src/components/Forms/Switch/index.stories.mdx:131]: remove extra space while writing code

File: example/storybook/src/components/Forms/Switch/index.stories.mdx [hunk 6]
```diff
@@ -94,6 +94,150 @@ The following are the lists of all the libraries and packages the component reli
...
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+             <VStack space="md">
+                   <Switch size="sm" isChecked={true}/>
+                   <Switch  isChecked={true}/>
```
Comment (amars29) [inline example/storybook/src/components/Forms/Switch/index.stories.mdx:200]: remove extra space

File: example/storybook/src/components/MediaAndIcons/Avatar/index.stories.mdx [hunk 7]
```diff
@@ -123,6 +124,376 @@ The following are the lists of all the libraries and packages the component reli
...
+            <Avatar.FallbackText>Sandeep Srivastava</Avatar.FallbackText>  
+          </Avatar>
+          <Avatar size="lg" bgColor="$cyan600" borderColor="white" borderWidth="$2">
+            <Avatar.FallbackText>Arlene KcCoy</Avatar.FallbackText>
+          </Avatar>
+          <Avatar size="lg" bgColor="$indigo600" borderColor="white" borderWidth="$2">
+            <Avatar.FallbackText>Rahul Sharma </Avatar.FallbackText>
+          </Avatar>
+          <Avatar size="lg" bgColor="$gray600" borderColor="white" borderWidth="$2">
+            <Avatar.FallbackText>+3 30</Avatar.FallbackText>
```
Comment (amars29) [inline example/storybook/src/components/MediaAndIcons/Avatar/index.stories.mdx:378]: Please correct the contect

File: example/storybook/src/components/Overlay/Menu/index.stories.mdx [hunk 8]
```diff
@@ -42,35 +42,37 @@ Following is the default implementation of the **Menu** component without any ad
...
-  <Menu.Backdrop />
-</Menu>
-`,
+        <Box h="$64">
+          <Menu
+            placement={"top"}
+            trigger={(triggerProps) => {
+              return (
+                <Button {...triggerProps} mt={200}>
+                  <Button.Text>
```
Comment (amars29) [inline example/storybook/src/components/Overlay/Menu/index.stories.mdx:51]: Icon shouldn't come under Button.Text (it should only contain text)

File: example/storybook/src/components/Overlay/Popover/index.stories.mdx [hunk 9]
```diff
@@ -39,61 +39,60 @@ Following is the default implementation of the **Popover** component without any
...
+          placement={'bottom'}
+          trigger={(triggerProps) => {
+            return (
+              <Pressable
+                bgColor={'$red500'}
+                borderRadius={'$md'}
+                mt='$12'
+                {...triggerProps}
+              >
+                <Text color={'white'} padding="$3">
```
Comment (amars29) [inline example/storybook/src/components/Overlay/Popover/index.stories.mdx:53]: remove parenthesis and directly write the string value in `color`

File: example/storybook/src/components/Feedback/Spinner/index.stories.mdx (hunk 2 above)
Comment (sandeepgeeky) [reply example/storybook/src/components/Feedback/Spinner/index.stories.mdx:154]: we cannot use token here it is not working

**Pull request closed without merging**